tests/tests.cpp

```diff
@@ -464,6 +482,21 @@ TEST_CASE("test BFS path medium", "[weight=1][part=2]")
     delete graph;
 }
 
+TEST_CASE("test cheapest path large", "[weight=1][part=2]")
```

Comment:
delete this...

Reply:
like the whole test.  Nisha and I concluded we don't need this because BFS can vary because it randomly chooses neighbors....